### ISSUE TITLE
Device list incorrectly spaced for CL devices lacking ADL

### ISSUE THREAD

**Issue by wizkid057** [luke-jr/bfgminer#92]
The device list has incorrect spacing for an OpenCL miner which lacks ADL.

```
--------------------------------------------------------------------------------
 [P]ool management [G]PU management [S]ettings [D]isplay options [Q]uit
 OCL 0:   1.8/  1.8/  0.0Mh/s | A:0 R:0 HW:0 U:0.00/m
 CPU 0:                | 210.0/205.9/  0.0kh/s | A:0 R:0 HW:0 U:0.00/m
 CPU 1:                | 283.1/293.4/  0.0kh/s | A:0 R:0 HW:0 U:0.00/m
--------------------------------------------------------------------------------
```

**Comment by luke-jr** [comment]
Fixed in 920166a

**Comment by luke-jr** [comment]
Not enough padding added

**Comment by luke-jr** [comment]
Fixed in #111
